zenodo/modules/records/views.py

```diff
@@ -440,12 +440,31 @@ def community_curation(record, user):
             accepted,
         )
 
+def get_reana_badge(record):
+    img_url = 'https://camo.githubusercontent.com/e7778200587ecd095b6d70c66bae6c09b3c813a37402f93894cc48446c9b155e/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f52756e2532306f6e2d5245414e412d666633333336'
+    for file in record.files:
+        if str(file["key"]) == "reana.yaml":
+            return {
+                'img_url': img_url,
+                'url': u'https://reana.cern.ch/run?from=url;uri=zenodo.org/{}/files/reana.yaml'.format(record.get('recid'))
```

Comment:
In the end, the URL will be something like this instead:

```
https://reana.cern.ch/launch?url=https://zenodo.org/{}/files/reana.yaml
```

The only required query string parameter is `url` so far. Other parameters accepted are `name` (workflow name) and `parameters` (encoded JSON object). E.g.

```
https://reana.cern.ch/launch?url=https://zenodo.org/xxx/files/reana.yaml&name=myzenodoanalysis&parameters=%7B%22events%22%3A%222000%22%7D
```

Reply:
Currently workflow `name` and `parameters` are discarded from the url.